src/common/Input.ts

```diff
@@ -22,15 +22,81 @@ type KeyDescription = Required<
   Pick<KeyDefinition, 'keyCode' | 'key' | 'text' | 'code' | 'location'>
 >;
 
+/**
+ * Keyboard provides an api for managing a virtual keyboard.
+ * The high level api is {@link Keyboard."type"},
```

Comment:
@TimvdLippe you have to quote the `type` here as it's a TSDoc keyword - that's why the link didn't work before. Bit obscure but ah well.